scripts/bump-executable-spec.sh

```diff
@@ -44,7 +44,10 @@ if [ $LEDGER_REPO_DIR ]; then
   pushd $LEDGER_REPO_DIR
   sed -i "s/tag: .*/tag: $EXEC_SPEC_COMMIT/" cabal.project
   sed -i "s/--sha256: .*/--sha256: sha256-0000000000000000000000000000000000000000000=/" cabal.project
+  sed -i "s/[^-]subdir/ --subdir/" cabal.project
+  sed -i 's/location: .*$/location: https:\/\/github.com\/Soupstraw\/exec-spec-temporary\.git/' cabal.project
```

Comment:
Leftover?

Reply:
I'm not sure what you're referring to. This script is probably only useful for me, but I'd like to keep it around just so I can test things out faster.

Reply:
The commented out section is my attempt to automate filling out the `sha256` field in the cardano ledger cabal project file, but I didn't get it working. I want to try to get it working at some point but right now this script is good enough.